Acumatica.RESTClient/BaseApi/ODataAPI.cs

```diff
@@ -12,136 +12,68 @@
 namespace Acumatica.RESTClient.Api
 {
 
-    public class ODataAPI
+    public class ODataAPI : BaseApi
     {
-        protected readonly string Username;
-        protected readonly string Password;
-        protected string BasePath;
+      
         protected ODataVersion Version;
         protected string Tenant;
         protected Token Token;
```

Comment:
I don't think we really need to store Token here as Token is already store in the configuration